src/nano_contracts/deserializer.ts

```diff
@@ -85,6 +102,18 @@ class Deserializer {
     return unpackToInt(4, true, value)[0];
   }
 
+  /**
+   * Deserialize amount value
+   *
+   * @param {value} Value to deserialize
+   *
+   * @memberof Deserializer
+   * @inner
+   */
+  toAmount(value: Buffer): OutputValueType {
+    return this.toInt(value);
```

Comment:
suggestion(non-blocking): It would be nice to have a simple test here, just to complete the coverage, although we know this will soon be made more complex by the increased limit of accepted values.

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/729/commits/02b94f0553285cef8c2ad3e222da8fb8d065ea9d